.github/workflows/check.yaml

```
@@ -0,0 +1,103 @@
+name: Tests
+
+on:
+  workflow_call:
+  workflow_dispatch:
+  pull_request:
+    types: [opened, synchronize, reopened]
+    branches: [main]
+    paths-ignore:
+      - "**.md"
+      - "**.rst"
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  lint:
+    name: Lint
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: ["3.8", "3.10"]
+    runs-on: ubuntu-latest
```

Is this file going to be managed by the automation?
```suggestion
    runs-on: ubuntu-22.04
```

---

Thanks - it's not currently managed by automation, but it will be eventually. :)